endpoints.go

```diff
@@ -144,9 +176,11 @@ var (
 	EndpointEmoji         = func(eID string) string { return EndpointCDN + "emojis/" + eID + ".png" }
 	EndpointEmojiAnimated = func(eID string) string { return EndpointCDN + "emojis/" + eID + ".gif" }
 
-	EndpointOauth2            = EndpointAPI + "oauth2/"
-	EndpointApplications      = EndpointOauth2 + "applications"
-	EndpointApplication       = func(aID string) string { return EndpointApplications + "/" + aID }
-	EndpointApplicationsBot   = func(aID string) string { return EndpointApplications + "/" + aID + "/bot" }
-	EndpointApplicationAssets = func(aID string) string { return EndpointApplications + "/" + aID + "/assets" }
+	EndpointOauth2                  = EndpointAPI + "oauth2/"
```

Comment:
Maybe use `OAuth2`, with uppercase `A`?

Reply:
Yes, I think `OAuth2` will be better. U can check over rep like https://github.com/golang/oauth2/blob/master/oauth2.go and all comment use `OAuth`